core/delegation/delegation_test.go

```diff
@@ -119,3 +120,18 @@ func TestFormatParse(t *testing.T) {
 
 	require.Equal(t, dlg.Link(), parsed.Link())
 }
+
+//go:embed delegationnonb.car
+var delegationnonb []byte
```

Comment:
Based on our conversation related to this, let's include a comment with the relevant JS code used to generate this file.